source/nodejs/adaptivecards-designer/src/open-sample-dialog.ts

```diff
@@ -171,6 +214,22 @@ export class OpenSampleDialog extends Dialog {
                 this._output = output;
                 this.close();
             }
+            itemElement.onNavigateToNextList = () => {
+                // Find the index of the current list
+                const currentIndex = this._listElements.indexOf(listElement);
```

Comment:
I assume currentIndex is always `>=0`, otherwise this code is problematic.
If listElement is not in the list, then currentIndex is -1.

Reply:
Good call - I'll add in a check here just to make sure nothing weird happens! But theoretically, yes the `currentIndex` should be `>=0` since we generate `_listElements` on line 208

Reply:
I added the check to make sure current index is not `-1` and included an upper bounds check.